src/grandorgue/combinations/GOSetter.cpp

```diff
@@ -135,111 +135,111 @@ enum {
-    {wxT("General25"), ID_SETTER_GENERAL24, true, true},
-    {wxT("General26"), ID_SETTER_GENERAL25, true, true},
-    {wxT("General27"), ID_SETTER_GENERAL26, true, true},
-    {wxT("General28"), ID_SETTER_GENERAL27, true, true},
-    {wxT("General29"), ID_SETTER_GENERAL28, true, true},
-    {wxT("General30"), ID_SETTER_GENERAL29, true, true},
-    {wxT("General31"), ID_SETTER_GENERAL30, true, true},
-    {wxT("General32"), ID_SETTER_GENERAL31, true, true},
-    {wxT("General33"), ID_SETTER_GENERAL32, true, true},
-    {wxT("General34"), ID_SETTER_GENERAL33, true, true},
-    {wxT("General35"), ID_SETTER_GENERAL34, true, true},
-    {wxT("General36"), ID_SETTER_GENERAL35, true, true},
-    {wxT("General37"), ID_SETTER_GENERAL36, true, true},
-    {wxT("General38"), ID_SETTER_GENERAL37, true, true},
-    {wxT("General39"), ID_SETTER_GENERAL38, true, true},
-    {wxT("General40"), ID_SETTER_GENERAL39, true, true},
-    {wxT("General41"), ID_SETTER_GENERAL40, true, true},
-    {wxT("General42"), ID_SETTER_GENERAL41, true, true},
-    {wxT("General43"), ID_SETTER_GENERAL42, true, true},
-    {wxT("General44"), ID_SETTER_GENERAL43, true, true},
-    {wxT("General45"), ID_SETTER_GENERAL44, true, true},
-    {wxT("General46"), ID_SETTER_GENERAL45, true, true},
-    {wxT("General47"), ID_SETTER_GENERAL46, true, true},
-    {wxT("General48"), ID_SETTER_GENERAL47, true, true},
-    {wxT("General49"), ID_SETTER_GENERAL48, true, true},
-    {wxT("General50"), ID_SETTER_GENERAL49, true, true},
-
-    {wxT("GeneralPrev"), ID_SETTER_GENERAL_PREV, true, true},
-    {wxT("GeneralNext"), ID_SETTER_GENERAL_NEXT, true, true},
-
-    {wxT("PitchP1"), ID_SETTER_PITCH_P1, true, true},
-    {wxT("PitchP10"), ID_SETTER_PITCH_P10, true, true},
-    {wxT("PitchP100"), ID_SETTER_PITCH_P100, true, true},
-    {wxT("PitchM1"), ID_SETTER_PITCH_M1, true, true},
-    {wxT("PitchM10"), ID_SETTER_PITCH_M10, true, true},
-    {wxT("PitchM100"), ID_SETTER_PITCH_M100, true, true},
-    {wxT("TemperamentPrev"), ID_SETTER_TEMPERAMENT_PREV, true, true},
-    {wxT("TemperamentNext"), ID_SETTER_TEMPERAMENT_NEXT, true, true},
-    {wxT("TransposeDown"), ID_SETTER_TRANSPOSE_DOWN, true, true},
-    {wxT("TransposeUp"), ID_SETTER_TRANSPOSE_UP, true, true},
-
-    {wxT("Save"), ID_SETTER_SAVE, true, true},
-    {wxT("OnState"), ID_SETTER_ON, false, true},
-
-    {wxT("CrescendoA"), ID_SETTER_CRESCENDO_A, true, true},
-    {wxT("CrescendoB"), ID_SETTER_CRESCENDO_B, true, true},
-    {wxT("CrescendoC"), ID_SETTER_CRESCENDO_C, true, true},
-    {wxT("CrescendoD"), ID_SETTER_CRESCENDO_D, true, true},
-    {wxT("CrescendoPrev"), ID_SETTER_CRESCENDO_PREV, true, true},
-    {wxT("CrescendoCurrent"), ID_SETTER_CRESCENDO_CURRENT, true, true},
-    {wxT("CrescendoNext"), ID_SETTER_CRESCENDO_NEXT, true, true},
-    {wxT("CrescendoOverride"), ID_SETTER_CRESCENDO_OVERRIDE, true, true},
-    {wxT(""), -1, false, false},
+    {wxT("Prev"), ID_SETTER_PREV, true, true, false},
```

Comment:
As I don't remember my c++ basis, I'm wondering if the dictionary can't be initialized with default values in order to simplify writings here ?

Reply:
This kind of initializers doesn't respect the default values. A code without the values cann't be compiled without errors.